composite/synth-index/src/index.ts

```diff
@@ -0,0 +1,7 @@
+import { expose, util } from '@chainlink/ea-bootstrap'
+import { makeExecute } from './adapter'
+import { makeConfig } from './config'
+
+const NAME = 'Synth Index'
```

Comment:
We should keep the naming convention of no spaces. We have also been capitalizing this in a lot of places.
```suggestion
const NAME = 'SYNTH-INDEX'
```